database/config_test.go

```diff
@@ -64,15 +65,15 @@ func TestTableInfoStore(t *testing.T) {
 
 		// Getting a non-existing TableInfo should not work.
 		_, err = tx.tableInfoStore.Get(tx, "unknown")
-		require.Equal(t, ErrTableNotFound, err)
+		require.Equal(t, fmt.Errorf("%w: %q", ErrTableNotFound, "unknown"), err)
 
 		// Deleting an existing TableInfo should work.
 		err = tx.tableInfoStore.Delete(tx, "foo1")
 		require.NoError(t, err)
 
 		// Deleting a non-existing TableInfo should not work.
 		err = tx.tableInfoStore.Delete(tx, "foo1")
-		require.Equal(t, ErrTableNotFound, err)
+		require.Equal(t, fmt.Errorf("%w: %q", ErrTableNotFound, "foo1"), err)
```

Comment:
```suggestion
		require.ErrorIs(t, err, ErrTableNotFound)
```

Reply:
Oh thanks, I will change that

Reply:
Hi,
I commited this suggestion but there is no require.ErrorIs and now the build is failing.

Is there something I didn't understand?
Thank you